KarhooSDK/Api/Response/PaymentProvider.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Provider.swift
+//  KarhooSDK
+//
+//  Copyright © 2020 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+
+public struct PaymentProvider : KarhooCodableModel {
```

Comment:
In Android, it is called [Provider](https://github.com/karhoo/karhoo-android-sdk/blob/develop/sdk/src/main/java/com/karhoo/sdk/api/model/Provider.kt), would be good to have consistency. I think it makes sense to call it PaymentProvider though. 

@jeremyauclert